SignalMessaging/utils/OWSMessagesBubbleImageFactory.swift

```diff
@@ -58,6 +57,32 @@ public class OWSMessagesBubbleImageFactory: NSObject {
         }
     }
 
+    public static let bubbleColorIncoming = UIColor.jsq_messageBubbleLightGray()!
+
+    public static let bubbleColorOutgoingUnsent = UIColor.gray
```

Comment:
We should dedupe and use these constants in the above JSQMessagesBubbleImage builders, or delete those other methods if they're no longer being used.

Reply:
Done. Once we've worked on message detail view, I hope to rename and refactor `OWSMessagesBubbleImageFactory`.